support/ebpf/ruby_tracer.ebpf.c

```diff
@@ -247,18 +257,20 @@ int unwind_ruby(struct pt_regs *ctx) {
     // the offset to running_ec.
 
     void *single_main_ractor = NULL;
-    if (bpf_probe_read_user(&single_main_ractor, sizeof(single_main_ractor),
-                            (void *)rubyinfo->current_ctx_ptr)) {
+    if (bpf_probe_read_user(
+            &single_main_ractor, sizeof(single_main_ractor), (void *)rubyinfo->current_ctx_ptr)) {
```

Comment:
8 spaces alignment seems to be wrong here.

Reply:
resolving, as this is no longer the case